service/src/main/java/org/cbioportal/service/CopyNumberEnrichmentService.java

```diff
@@ -4,11 +4,11 @@
 import org.cbioportal.service.exception.MolecularProfileNotFoundException;
 
 import java.util.List;
+import org.cbioportal.model.Entity;
 
 public interface CopyNumberEnrichmentService {
-    
-    List<AlterationEnrichment> getCopyNumberEnrichments(String molecularProfileId, List<String> alteredIds, 
-                                                        List<String> unalteredIds, List<Integer> alterationTypes, 
-                                                        String enrichmentType) 
+
+    List<AlterationEnrichment> getCopyNumberEnrichments(List<Entity> set1, List<Entity> set2,
```

Comment:
personally I think set is alright because it's clear in the context of what the API is what those sets are, but making the names more descriptive can never hurt